File: Data Structures & Algorithms/duplicate-integer/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {boolean}
     */
    hasDuplicate(nums) {
        let numSet = new Set()

        for(let i = 0; i < nums.length; i++) {
            let curr = nums[i]
            if(numSet.has(curr)) {
                return true
            } else {
                numSet.add(curr)
            }
        }

        return false;
    }
}

File: Data Structures & Algorithms/missing-number/submission-0.java
class Solution {
    public int missingNumber(int[] nums) {
        int size = nums.length;
        int shouldBe = size * (size+1) / 2;

        int currSum = 0;
        for(int i = 0; i < size; i++) {
            currSum += nums[i];
        }

        return shouldBe - currSum;
    }
}
